### PR TITLE
New homepage

### LINKED ISSUE #1327: Design new docs homepage
Need to create a new homepage that leads users to v1 and v2 content.

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1327 

image

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -1,14 +1,89 @@
...
+    <div class="group-wrapper">
+      <div class="card quarter">
+        <h2>Enterprise offerings</h2>
+        <p>Enterprise tools to meet your business needs.</p>
+        <p><a href="https://www.influxdata.com/contact-sales/">Contact Sales</a></p>
+      </div>
+      <div id="tick-cards" class="three-quarters row">
+        <div class="card influxdb">
+          <h2><a href="/enterprise_influxdb/{{ $enterpriseVersion }}/">InfluxDB Enterprise <span class="version">{{ $enterpriseVersion }}</span></a></h2>
+          <p>A scalable InfluxDB cluster designed for heavy workloads. Run InfluxDB Enterprise on your own infrastructure — on-premise, private cloud, public cloud, or on the edge.</p>
```
Comment (pierwill) [inline layouts/index.html:81]: on-premises

File: layouts/index.html [hunk 2]
```diff
@@ -1,14 +1,89 @@
...
+{{ partial "topnav.html" . }}
+
+<div class="home">
+
+  <div class="section hero">
+    {{ partial "svgs/homepage-hero.svg" }}
+    <div class="half">
+      <h2>InfluxDB Cloud & <br />InfluxDB {{ $influxdbVersionV2 }} OSS</h2>
+      <p>
+        The most powerful time series database designed to handle high write and query loads.
```
Comment (kelseiv) [inline layouts/index.html:18]: We could split first sentence for emphasis:
The most powerful...Designed to handle

File: layouts/index.html [hunk 3]
```diff
@@ -1,14 +1,89 @@
...
+              spellcheck="false"
+              dir="auto"
+              placeholder='Search the docs'>
+    </div>
+  </div>
+
+  <div id="tick-stack" class="section row">
+    <div class="group-wrapper">
+      <div id="platform" class="card quarter">
+        <h2><a href="/platform/">InfluxData TICK Stack</a></h2>
```
Comment (kelseiv) [inline layouts/index.html:42]: stack

File: layouts/index.html [hunk 4]
```diff
@@ -1,14 +1,89 @@
...
+      <div class="card quarter">
+        <h2>Enterprise offerings</h2>
+        <p>Enterprise tools to meet your business needs.</p>
+        <p><a href="https://www.influxdata.com/contact-sales/">Contact Sales</a></p>
+      </div>
+      <div id="tick-cards" class="three-quarters row">
+        <div class="card influxdb">
+          <h2><a href="/enterprise_influxdb/{{ $enterpriseVersion }}/">InfluxDB Enterprise <span class="version">{{ $enterpriseVersion }}</span></a></h2>
+          <p>A scalable InfluxDB cluster designed for heavy workloads. Run InfluxDB Enterprise on your own infrastructure — on-premise, private cloud, public cloud, or on the edge.</p>
+          <p><a href="/enterprise_influxdb/{{ $enterpriseVersion }}/">InfluxDB Enterprise documentation</a></p>
```
Comment (kelseiv) [inline layouts/index.html:82]: View doc...

File: layouts/index.html [hunk 5]
```diff
@@ -1,14 +1,89 @@
...
+      <div id="tick-cards" class="three-quarters row">
+        <div class="card telegraf">
+          <h2><a href="/telegraf/{{ $telegrafVersion }}/">Telegraf <span class="version">{{ $telegrafVersion }}</span></a></h2>
+          <p>Collect metrics from stacks, sensors and systems.</p>
+          <p><a href="/telegraf/{{ $telegrafVersion }}/">Telegraf documentation</a></p>
+        </div>
+        <div class="card influxdb">
+          <h2><a href="/influxdb/{{ $influxdbVersionV1 }}/">InfluxDB <span class="version">{{ $influxdbVersionV1 }}</span></a></h2>
+          <p>Write and query time series data.</p>
+          <p><a href="/influxdb/{{ $influxdbVersionV1 }}/">InfluxDB  documentation</a></p>
```
Comment (kelseiv) [inline layouts/index.html:55]: View doc...

File: layouts/index.html [hunk 6]
```diff
@@ -1,14 +1,89 @@
...
+        </div>
+        <div class="card chronograf">
+          <h2><a href="/chronograf/{{ $chronografVersion }}/">Chronograf <span class="version">{{ $chronografVersion }}</span></a></h2>
+          <p>Visualize and manage time series data.</p>
+          <p><a href="/chronograf/{{ $chronografVersion }}/">Chronograf documentation</a></p>
```
Comment (kelseiv) [inline layouts/index.html:60]: View doc...

File: layouts/index.html [hunk 7]
```diff
@@ -1,14 +1,89 @@
...
+        </div>
+        <div class="card kapacitor">
+          <h2><a href="/kapacitor/{{ $kapacitorVersion }}/">Kapacitor <span class="version">{{ $kapacitorVersion }}</span></a></h2>
+          <p>Process and alert on time series data.</p>
+          <p><a href="/kapacitor/{{ $kapacitorVersion }}/">Kapacitor documentation</a></p>
```
Comment (kelseiv) [inline layouts/index.html:65]: View doc...

File: layouts/index.html (hunk 1 above)
Comment (pierwill) [reply layouts/index.html:81]: Maybe "at the edge".

Comment (kelseiv) [reply layouts/index.html:81]: infrastructure—on-

**Pull request merged**